update Performance index page and restructure (#3466)
getsentry/sentry-docs: src/docs/product/performance/filters-display.mdx

---
title: "Filters & Display"
sidebar_order: 20
redirect_from:
  - /product/performance/display/

> description: "Filter the information displayed on the Performance page using search or various page-level display filters."

Reviewer: ```suggestion
description: "Manage the information on the Performance page using search or page-level display filters to quickly identify performance issues."
```

---

> There are several ways you can filter the information displayed on the **Performance** page, by using search or various page-level display filters.

Reviewer: ```suggestion
You can filter the information displayed on the **Performance** page by searching or using page-level display filters.
```

## Transaction and Trend Views

> Using the “By Transaction/By Trend” toggle, you can set the overall display of the page. By default, the page is set to view transactions and the different ways that you can filter and set the display of transactions is described in the rest of this page. Learn more about viewing transactions by trend in the [Trends View documentation](/product/performance/trends/).

Reviewer: ```suggestion
The page default is to view transactions, but you can use the “By Transaction/By Trend” toggle to change the overall display of the page. The different ways you can filter and set the display of transactions is described fully on this page. To learn more about viewing transactions by trend, see our [Trends View documentation](/product/performance/trends/).
```

## Search Condition Filters

The search bar operates similarly to the one in the Discover [Query Builder](/product/discover-queries/query-builder/#filter-by-search-conditions). For example, you can refine your transaction search to a specific release by using `release:` as a key field and assigning the version as a value.

## Display Filter

> The "Display" filter is next to the search bar. It automatically selects a set of [metrics](/product/performance/metrics/) and table columns based on the platform of your project. For example, a JavaScript project defaults to a Frontend (Pageload) display. You can change the selected display, though note that some modes are not applicable to the platform.

Reviewer 2: Could also be `not applicable for the selected platform`. The current wording just feels off.
```suggestion
The "Display" filter is next to the search bar. It automatically selects a set of [metrics](/product/performance/metrics/) and table columns based on the platform of your project. For example, a JavaScript project defaults to a Frontend (Pageload) display. You can change the selected display, though note that some modes are not applicable to all platforms.
```

> The "Display" filter can be found next to the search bar. This  automatically selects a set of [metrics](/product/performance/metrics/) and table columns based on the platform of your project. For example, a JavaScript project defaults to a Frontend (Pageload) display. You can change the selected display, but some modes may not be applicable to the platform.

Reviewer: ```suggestion
The "Display" filter is next to the search bar. It automatically selects a set of [metrics](/product/performance/metrics/) and table columns based on the platform of your project. For example, a JavaScript project defaults to a Frontend (Pageload) display. You can change the selected display, though note that some modes are not applicable to the platform.
```

### All Transactions

> When "All Transactions" is selected, the table includes all frontend and backend transactions.

Reviewer: ```suggestion
When you select "All Transactions", the table includes all frontend and backend transactions.
```

![Performance Homepage with Display: All](00_performance_landing.png)

### Frontend (Pageload)

"Frontend (Pageload)" puts an emphasis on [Web Vitals](/product/performance/web-vitals/). The transactions are automatically filtered to only include `transaction.op:pageload`. This generally corresponds to all browser page transitions that are not handled internally in SPA frameworks, such as React or Vue.

![Performance Homepage with Display: Frontend (Pageload)](01_performance_landing.png)

### Frontend (Other)

"Frontend (Other)" includes any other frontend transaction that is not a pageload. The filter `transaction.op:!pageload` automatically gets added. This generally corresponds to page transitions handled internally in SPA frameworks, such as React or Vue. Note that if you've selected a backend project, this mode also displays backend transactions.

### Backend

"Backend" puts an emphasis on duration, throughput, failure rate, and Apdex. Note that if you've selected a frontend project, this mode also displays all frontend transactions.

![Performance Homepage with Display: Backend](02_performance_landing.png)

## Graphs

> You can compare graphs side by side to find any relevant correlations. For example, if an Apdex score dips significantly during a given time period, you'll be able to visualize whether throughput spiked in the same time interval in a "Transactions per Minute" graph. The global header and search conditions will also allow you to filter these graphs even more. The display options are listed below:

Reviewer: ```suggestion
You can compare graphs side by side to find any relevant correlations. For example, if an Apdex score dips significantly during a given time period, you'll be able to visualize whether throughput spiked in the same time interval in a "Transactions per Minute" graph. The global header and search conditions will also allow you to further filter these graphs:
```

- [Apdex](/product/performance/metrics/#apdex)
- [Transactions Per Minute (TPM)](/product/performance/metrics/#throughput-total-tpm-tps)
- [Failure Rate](/product/performance/metrics/#failure-rate)
- [p50 Duration](/product/performance/metrics/#p50-threshold)
- [p95 Duration](/product/performance/metrics/#p95-threshold)
- [p99 Duration](/product/performance/metrics/#p99-threshold)

## Transaction Table

> The table of transactions provides you with a list of the most frequent transactions based on the current page filters and display options. By default, the table is sorted by transactions per minute, with the most frequent transactions displayed first. The table may be filtered further based on global header and search condition filters.

Reviewer: ```suggestion
The table of transactions provides a list of the most frequent transactions based on the current page filters and display options. By default, the table is sorted by transactions per minute, with the most frequent transactions displayed first. The table may be filtered further based on global header and search condition filters.
```

> The [Display Filter](#display-filter) will automatically set what appears in the table columns. Here are a few of the common columns shown in the tables.

Reviewer: ```suggestion
The [Display Filter](#display-filter) controls what displays in the table columns, such as:
```

> The [Display Filter](/product/performance/display-filters/#display-filter) will automatically set what appears in the table columns. Here are a few of the common columns shown in the tables.

Author: ```suggestion
The [Display Filter](/product/performance/filters-display/#display-filter) will automatically set what appears in the table columns. Here are a few of the common columns shown in the tables.
```
